jwst/model_blender/blendrules.py

```diff
@@ -81,6 +83,29 @@ def last(items):
         return items[-1]
     return None
 
+def mindate(items):
+    """Return the minimum time from a list of time strings."""
+    time_list = Time(items)
+    return time_list.min()
+
+def maxdate(items):
+    """Return the maximum time from a list of time strings."""
+    time_list = Time(items)
+    return time_list.max()
+
+def mintime(items):
+    times = [_isotime(time_str) for time_str in items]
+    return min(times).isoformat()
+
+def maxtime(items):
+    times = [_isotime(time_str) for time_str in items]
+    return max(times).isoformat()
```

Comment:
It would be helpful for future readability if the docstrings in the rules mention which keywords the rules are used for by adding `Parameters` section. This information can currently be found by looking at the core schema which isn't convenient.

Reply:
Given that the 'first' one is the default for all keywords, I am not sure how practical this would be.  Also, you would then be introducing a situation where the docstrings can get out of sync with the schema if one is updated and not the other (I realize this never happens, but... ;-) ).   Not saying this is a bad idea, just that it comes with a risk.